api/src/test/resources/underlays/nautical.pbtext

```diff
@@ -0,0 +1,305 @@
+# proto-file: proto/bio/terra/tanagra/underlay/underlay.proto
```

Comment:
Given the length of this simple example, it seems possible that we may eventually want to support splitting across multiple files

Reply:
Yep. I could imagine splitting on the same lines as the protos (entities/tables/mapping), but at the same time I'd like to wait to have more experience with building/generating these and thinking about tooling we might like to have before getting too much into ergonomics.